openVirtualKeyboard.js

```diff
@@ -0,0 +1,35 @@
+
+/**
+ * Overview
+ *  - This is utility to open virtual keyboard in mobile.
+ * 
+ * Usage pattern:
+ *  - Import 
+ *    - import { openVirtualKeyboard } from '@dreamworld/web-util.js'
+ *  - Use
+ *    - openVirtualKeyboard();
+ *
+ */
+
+class OpenVirtualKeyboard {
```

Comment:
Here, we don't need to create a class, it's redundant from my point of view and makes this logic hard to understand.